src/cache.js

```diff
@@ -266,6 +268,26 @@ class Cache {
       this._cache = {};
       this._saveLastCleared();
   }
+
+  refresh(xmlDoc, refreshedtype) {
+    var clearCache = DomUtil.getAttributeAsString(xmlDoc, "emptyCache");
+    if (clearCache == "true") {
+      console.log("Clear cache");
```

Comment:
Remove all the console.log which will pollute the output. I do not have a proper logging mechanism in the SDK right now.